var/spack/repos/builtin/packages/dihydrogen/package.py

```diff
@@ -62,8 +62,9 @@ class Dihydrogen(CMakePackage, CudaPackage, ROCmPackage):
 
     # Specify the correct version of Aluminum
     depends_on('aluminum@0.4:0.4.99', when='@0.1:0.1.99 +al')
-    depends_on('aluminum@0.5.0:0.7.99', when='@0.2.0:0.2.1 +al')
-    depends_on('aluminum@0.5.0:', when='@:0.0,0.2.1: +al')
+    depends_on('aluminum@0.5.0:0.5.99', when='@0.2.0 +al')
+    depends_on('aluminum@0.7.0:0.7.99', when='@0.2.1 +al')
+    depends_on('aluminum@0.7.0:', when='@:0.0,0.2.1: +al')
```

Comment:
There are no listed versions of dihydrogen that are < 0.0, what is the `@:0.0` intended to resolve here?

Reply:
Arbitrary labels, that are not `develop`, `master`